core/domain/user_services.py

```diff
@@ -1118,6 +1241,14 @@ def mark_user_for_deletion(user_id):
     user_settings = get_user_settings(user_id, strict=True)
     user_settings.deleted = True
     _save_user_settings(user_settings)
+    user_auth = _get_user_auth_from_model(
+        user_models.UserAuthModel.get_by_id(user_id))
+
+    # TODO(#10178): Remove the if condition below when UserAuthModel will
```

Comment:
when UserAuthModel will be --> once UserAuthModel is

Reply:
Done.